src/ui/common/src/components/workflows/edges/AqueductQuadratic.tsx

```diff
@@ -1,23 +1,8 @@
 import React from 'react';
+import { EdgeProps } from 'react-flow-renderer';
 import {theme} from "../../../styles/theme/theme";
 
-type AqueductQuadraticProps = {
```

Comment:
These are no longer needed and we can just use the built in EdgeProps type that comes with react-flow.